src/system.c

```diff
@@ -1707,37 +1708,48 @@ static Int preAllocAmount;
-  { 'n', "", forceLineEditing, 0, 0}, /* prob library */
-  { 'o', "", storeMemory2, &SyStorMax, 1 }, /* library with new interface */
-  { 'p', "", toggle, &SyWindow, 0 }, /* ?? */
-  { 'q', "", toggle, &SyQuiet, 0 }, /* ?? */
+    { 'B', "architecture", storeString, &SyArchitecture,
+      1 }, /* default architecture needs to be passed from kernel
+                 to library. Might be needed for autoload of compiled files */
+    { 'C', "", processCompilerArgs, 0, 4 }, /* must handle in kernel */
+    { 'D', "debug-loading", toggle, &SyDebugLoading,
+      0 }, /* must handle in kernel */
+    { 'K', "maximal-workspace", storeMemory2, &SyStorKill,
+      1 }, /* could handle from library with new interface */
+    { 'L', "", storeString, &SyRestoring, 1 }, /* must be handled in kernel */
+    { 'M', "", toggle, &SyUseModule, 0 },      /* must be handled in kernel */
+    { 'R', "", unsetString, &SyRestoring, 0 }, /* kernel */
+    { 'a', "", storeMemory, &preAllocAmount,
+      1 },                           /* kernel -- is this still useful */
+    { 'e', "", toggle, &SyCTRD, 0 }, /* kernel */
+    { 'f', "", forceLineEditing, (void *)2, 0 }, /* probably library now */
+    { 'E', "", toggle, &SyUseReadline, 0 },      /* kernel */
+    { 'l', "roots", setGapRootPath, 0, 1 },      /* kernel */
+    { 'm', "", storeMemory2, &SyStorMin, 1 },    /* kernel */
+    { 'r', "", toggle, &IgnoreGapRC, 0 },        /* kernel */
+    { 's', "", storeMemory, &SyAllocPool, 1 },   /* kernel */
+    { 'n', "", forceLineEditing, 0, 0 },         /* prob library */
+    { 'o', "", storeMemory2, &SyStorMax, 1 }, /* library with new interface */
+    { 'p', "", toggle, &SyWindow, 0 },        /* ?? */
+    { 'q', "", toggle, &SyQuiet, 0 },         /* ?? */
 #ifdef HPCGAP
-  { 'S', "", toggle, &ThreadUI, 0 }, /* Thread UI */
-  { 'Z', "", toggle, &DeadlockCheck, 0 }, /* Thread UI */
-  { 'P', "", storePosInteger, &SyNumProcessors, 1 }, /* Thread UI */
-  { 'G', "", storePosInteger, &SyNumGCThreads, 1 }, /* Thread UI */
+    { 'S', "", toggle, &ThreadUI, 0 },                 /* Thread UI */
+    { 'Z', "", toggle, &DeadlockCheck, 0 },            /* Thread UI */
+    { 'P', "", storePosInteger, &SyNumProcessors, 1 }, /* Thread UI */
+    { 'G', "", storePosInteger, &SyNumGCThreads, 1 },  /* Thread UI */
 #endif
-  /* The following three options must be handled in the kernel so they happen early enough */
-  { 0  , "prof", enableProfilingAtStartup, 0, 1},    /* enable profiling at startup */
-  { 0  , "cover", enableCodeCoverageAtStartup, 0, 1}, /* enable code coverage at startup */
-  { 0  , "quitonbreak", toggle, &SyQuitOnBreak, 0}, /* Quit GAP if we enter the break loop */
-  { 0, "", 0, 0, 0}};
+    /* The following three options must be handled in the kernel so they
+       happen early enough */
+    { 0, "prof", enableProfilingAtStartup, 0,
+      1 }, /* enable profiling at startup */
+    { 0, "memprof", enableMemoryProfilingAtStartup, 0,
```

Comment:
Reformatting all of `options` makes it difficult to spot what you actually changed :/